AUTOCONFIG/AutoconfigHandler.php

```diff
@@ -0,0 +1,1284 @@
+            'documentation_desc'		=> pacol( 1, 		1,      1,      'text', 'Autoconfig_documentation_desc'				, ''                                , '' ),
+            'webmail'					=> pacol( 1, 		1,      1,      'text', 'Autoconfig_webmail'				, ''                                , '' ),
+            'webmail_login_page'		=> pacol( 1, 		1,      1,      'text', 'Autoconfig_webmail_login_page'				, ''                                , '' ),
+            'webmail_login_page_info'	=> pacol( 1, 		1,      1,      'text', 'Autoconfig_webmail_login_page_info'				, ''                                , '' ),
+            'lp_info_url'				=> pacol( 1, 		1,      1,      'text', 'Autoconfig_lp_info_url'				, ''                                , '' ),
+            'lp_info_username'			=> pacol( 1, 		1,      1,      'text', 'Autoconfig_lp_info_username'				, ''                                , '' ),
+            'lp_info_username_field_id'	=> pacol( 1, 		1,      1,      'text', 'Autoconfig_lp_info_username_field_id'				, ''                                , '' ),
+            'lp_info_username_field_name'	=> pacol( 1, 		1,      1,      'text', 'Autoconfig_lp_info_username_field_name'				, ''                                , '' ),
+            'lp_info_login_button_id'	=> pacol( 1, 		1,      1,      'text', 'Autoconfig_lp_info_login_button_id'				, ''                                , '' ),
+            'lp_info_login_button_name'	=> pacol( 1, 		1,      1,      'text', 'Autoconfig_lp_info_login_button_name'				, ''                                , '' ),
+            // Mac Mail specific fields
+            'account_name'				=> pacol( 1, 		1,      1,      'text', 'Autoconfig_account_name'				, ''                                , '' ),
+            // Typically 'email'
+            'account_type'				=> pacol( 1, 		1,      1,      'text', 'Autoconfig_account_type'				, ''                                , '' ),
+            'email'						=> pacol( 1, 		1,      1,      'text', 'Autoconfig_email'				, ''                                , '' ),
+            'ssl_enabled'				=> pacol( 1, 		1,      1,      'boolean', 'Autoconfig_ssl'				, ''                                , '' ),
+            // Will be empty obviously unless the user enters it in the form
+            'password'					=> pacol( 1, 		1,      1,      'text', 'Autoconfig_password'				, ''                                , '' ),
+            // Used for payload_description
+            'description'				=> pacol( 1, 		1,      1,      'text', 'Autoconfig_description'				, ''                                , '' ),
+            'organisation'				=> pacol( 1, 		1,      1,      'text', 'Autoconfig_organisation'				, ''                                , '' ),
+            // regular account, or Microsoft Exchange
+            'type'						=> pacol( 1, 		1,      1,      'text', 'Autoconfig_type'				, ''                                , '' ),
```

Comment:
you've defined the 'type' column twice....

Reply:
Actually, I did not really understand the purpose of this `initStruct()`, and am not using it. Yes, the column appeared twice and this is a mistake. `type` is used as a property of the overall `autoconfig` and also as a property of the hosts `autoconfig`. My bad.